app/assets/components/service_update/view.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class ServiceUpdate::View < GovukComponent::Base
```

Comment:
I'm not that familiar with these Gov components but why do we put them in the assets folder? Why can't we follow the `view component` pattern and have them in `app/components`?